backend/kesaseteli/applications/models.py

```diff
@@ -1,11 +1,18 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 from encrypted_fields.fields import EncryptedCharField
-from shared.models.abstract_models import HistoricalModel, UUIDModel
+from shared.models.abstract_models import HistoricalModel, TimeStampedModel, UUIDModel
 
-from applications.enums import ApplicationStatus
+from applications.enums import ApplicationStatus, AttachmentType
 from companies.models import Company
 
+ATTACHMENT_CONTENT_TYPE_CHOICES = (
+    ("application/pdf", "pdf"),
+    ("image/png", "png"),
+    ("image/jpeg", "jpeg"),
+    ("image/jpg", "jpg"),
```

Comment:
would image/jpg ever be used? I understand it's non-standard

Reply:
Yea, with some further investigation this should never occur. I'll remove it.